src/Microsoft.DotNet.Interactive.Tests/LanguageServices/SignatureHelpTests.cs

```diff
@@ -81,6 +82,35 @@ public async Task signature_help_can_handle_language_switching_and_offsets()
                 .Contain(sigInfo => sigInfo.Label == "void Console.WriteLine()");
         }
 
+        [Theory]
+        [InlineData(Language.CSharp, "System.Environment.GetEnvironmentVariable($$", 0, "Retrieves the value of an environment variable from the current process")]
+        public async Task signature_help_can_return_doc_comments_from_bcl_types(Language language, string markupCode, int activeSignature, string expectedDocumentationSubstring)
+        {
+            using var kernel = CreateKernel(language);
+
+            MarkupTestFile.GetLineAndColumn(markupCode, out var code, out var line, out var character);
+            await kernel.SendAsync(new RequestSignatureHelp(code, new LinePosition(line, character)));
+
+            KernelEvents
+                .Should()
+                .ContainSingle<SignatureHelpProduced>()
+                .Which
+                .Signatures
+                .Should()
+                .HaveCountGreaterThan(activeSignature)
+                .And
+                .Subject
+                .LastOrDefault()
+                .Should()
+                .NotBeNull()
+                .And
+                .Subject
+                .As<SignatureInformation>()
+                .Documentation.Value
+                .Should()
+                .Contain(expectedDocumentationSubstring);
```

Comment:
`assertion.Should().BeReallyLong()`

Reply:
I'll try to add a few GUIDs to make it conform to corporate standards.